packages/components/src/templates/next/layouts/Homepage/Homepage.stories.tsx

```diff
@@ -30,6 +31,24 @@ const meta: Meta<typeof Homepage> = {
     themes: {
       themeOverride: "Isomer Next",
     },
+    mockData: [
+      {
+        url: "https://jsonplaceholder.com/muis_prayers_time",
+        method: "GET",
+        status: 200,
+        response: {
+          [getSingaporeDateYYYYMMDD()]: {
+            hijri_date: "17 Jamadilawal 1442H",
```

Comment:
nit: can standardise to hijriDate to use camelCase as default

Reply:
addressed in https://github.com/opengovsg/isomer/pull/902/commits/55b01825d5ccef052f2b1bdce47f8f3dbf0aae88